libraries/AP_HAL_SITL/SITL_State.h

```diff
@@ -170,6 +171,9 @@ class HALSITL::SITL_State {
     bool _use_fg_view;
     
     const char *_fdm_address;
+    const char *_simulator_address;
+    int _simulator_port_in;
+    int _simulator_port_out;
```

Comment:
yep you are right , the scope could be reduce to a local variable in SITL_State::_parse_command_line . I will correct that